### DISCUSSION TITLE
How to specify sliding window averages' ranges in the units of the field

### DISCUSSION THREAD

**Discussion by OpenFoam-User** [johnkerl/miller#1454] [Q&A]
According to documentation, to use the sliding window averages we need to use this syntax:

    mlr --csv step -a slwin_m_n -f <name of the field> filename.csv

Where `m` and `n` are integers used to specify the length of the window `m` records back and `n` records forward. That's perfectly fine for simple data.

**My question**:  
Is it possible to specify the length of the sliding window in the units of the field? For example if the field is time in seconds,  I want to do the average over a window of length `0.5` seconds back and `0.7` seconds forward. 

Thank you

**Comment by johnkerl** [comment]
@OpenFoam-User the sliding window is indeed in units of row numbers only.
